src/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -149,4 +150,16 @@ class CSVUtils {
     static CSVPrinter getCSVPrinter() {
         return new CSVPrinter(new StringBuilder(), CSVFormat.DEFAULT)
     }
+
+    static String addBOMToCSVString(String csvString) {
+        return '\uFEFF' + csvString
+    }
+
+    static String detectCSVCharset(File file) {
```

Comment:
In `ProductController`, there are two invocations of this method where, if it returns nothing, a default of `'MacRoman'` is applied. I wonder if that default should be specified here. Also, I imagine that most OB users are _not_ formatting their files on a Mac; would `'WINDOWS-1252'` be a better default?